Add config for ASUS Vivobook X580VD

Ensyell opened the pull request.

hirschmann commented:
Thank you very much for your effort!

There is just one small problem in your config:
The last two (up)thresholds are higher than the critical temperature setting.
Please, either adapt the critical temperature setting to the thresholds, or vice versa.

Ensyell commented:
Hi, 
you're welcome!
I changed it and added a threshold to start with a low rpm. I tested it these days and it works well, considering that the idle temperature of this pc is 40°C. Asus made this PC with two 2700 rpm fans at idle (they start turning at 36°C), this program is genial, I have a silent pc and a lot more battery now!

Thank you.

[ASUS Vivobook X580VD.zip](https://github.com/hirschmann/nbfc/files/1462399/ASUS.Vivobook.X580VD.zip)

hirschmann commented:
Thanks for the update! :+1:
